ocs_ci/framework/main.py

```diff
@@ -68,6 +68,8 @@ def init_ocsci_conf(arguments=None):
     parser.add_argument("--flexy-env-file", default="", help="Path to flexy env file")
     args, unknown = parser.parse_known_args(args=arguments)
     ocs_version = args.ocs_version
+    if framework.config.ENV_DATA["platform"].lower() == "openshiftdedicated":
```

Comment:
Can you please explain why we are setting ocs_version here to None? Maybe worth to add it also as comment here to explain why.

Anyway, later on we should load the ocs-config for specific version (non framework default) so if someone use --ocs-version parameter then we should load specific yaml file on lines 83-87.

I think that we should still load that version specific file no?  If I understand correctly user cannot actually pick the version of OCS and then they will just get what is available behind the scenes. But I think that many of tests just rely on version and we should have ability to set which version it's there no?

Reply:
passing --ocs-version in run-ci overrides the `ocs_csv_channel` and thus causing issues in `get_ocs_csv()` execution since the `ocs_csv_channel` in openshfit dedicated is "alpha". The above changes was to avoid the override of `ocs_csv_channel` and wasn't working as expected . I agree that making ocs_version to None will not actually pick the version of OCS and thus as an alternative approach extracted OCS CSV from the available CSVs instead of pulling from packagemanifests in `get_ocs_csv()`.